k2/torch/csrc/fsa_class.cu

```diff
@@ -109,8 +106,9 @@ RaggedArc RaggedArc::FromUnaryFunctionRagged(RaggedArc &src,
         auto filler_scalar = torch::tensor(
             filler, torch::dtype(torch::kInt32).device(value.device()));
         value = torch::where(masking, value, filler_scalar);
-        auto new_value = arc_map_any.Index(value, py::int_(filler));
-        dest.SetAttr(iter.first, new_value.RemoveValuesEq(py::int_(filler)));
+        auto new_value = arc_map_any.Index(value, torch::IValue(filler));
```

Comment:
You can convert `filler t`o an IValue in line 101 so that you don't need to convert it again in the following.